packages/ai-ide/src/browser/ai-configuration/language-model-renderer.tsx

```diff
@@ -79,76 +79,80 @@ export const LanguageModelRenderer: React.FC<LanguageModelSettingsProps> = (
             const resolvedModel = isAlias ? resolvedAliasModels[requirement.identifier] : undefined;
             return (
-                <React.Fragment key={index}>
-                    <div className="ai-alias-evaluates-to-container">
-                        <strong>{nls.localize('theia/ai/core/languageModelRenderer/purpose', 'Purpose')}:</strong> {requirement.purpose}
+                <div key={index} className="ai-llm-requirement-item">
+                    <div className="ai-configuration-value-row">
+                        <span className="ai-configuration-value-row-label">{nls.localize('theia/ai/core/languageModelRenderer/purpose', 'Purpose')}:</span>
+                        <span className="ai-configuration-value-row-value">{requirement.purpose}</span>
                     </div>
-                    <div>
-                        <div className="ai-alias-evaluates-to-container">
-                            <label
-                                className="theia-header no-select"
-                                htmlFor={`model-select-${agent.id}`}>
-                                {nls.localize('theia/ai/core/languageModelRenderer/languageModel', 'Language Model') + ': '}
-                            </label>
-                            <select
-                                className="theia-select"
-                                id={`model-select-${agent.id}-${requirement.purpose}`}
-                                value={requirement.identifier}
-                                onChange={event => onSelectedModelChange(requirement.purpose, event)}
-                            >
-                                <option value=""></option>
-                                {/* Aliases first, then languange models */}
-                                {aliases?.sort((a, b) => a.id.localeCompare(b.id)).map(alias => (
-                                    <option key={`alias/${alias.id}`} value={alias.id} className='ai-language-model-item-ready'>
-                                        {nls.localize('theia/ai/core/languageModelRenderer/alias', '[alias] {0}', alias.id)}
+                    <div className="ai-configuration-value-row">
+                        <label
+                            className="ai-configuration-value-row-label"
+                            style={{ lineHeight: '1.4' }}
+                            htmlFor={`model-select-${agent.id}-${requirement.purpose}`}>
+                            {nls.localize('theia/ai/core/languageModelRenderer/languageModel', 'Language Model')}:
+                        </label>
+                        <select
+                            className="theia-select ai-configuration-value-row-value"
+                            id={`model-select-${agent.id}-${requirement.purpose}`}
+                            value={requirement.identifier}
+                            onChange={event => onSelectedModelChange(requirement.purpose, event)}
+                        >
```

Comment:
This select box takes the whole remaining width and this feels a bit off, would be nice if we can use the same width/styling we use in the model aliases tab for example:

<img width="465" height="224" alt="Image" src="https://github.com/user-attachments/assets/49d03283-38cb-4020-b618-ec5dd07874f4" />